docs/src/features.md

```diff
@@ -0,0 +1,136 @@
+
+The standard method for reducing the memory requirements of a simulation in exchange for greater time complexity is known as slicing. This method works by fixing the value of selected indices in the network to decompose the contraction of the network into the sum of contractions of smaller networks. For more details on this method see the section on slicing in [this](https://arxiv.org/pdf/2005.06787.pdf) article.
+
+For a given contraction plan for a tensor network, `QXTools` uses [Shutski's tree-trimming](https://journals.aps.org/pra/abstract/10.1103/PhysRevA.102.062614), which is implemented in the package `QXGraphDecompositions` to select efficient indices of a network to slice. 
+
+To find a contraction plan for a quantum circuit using flowcutter and then slice a specific number of indices using Shutski's tree trimming method, the convenience function `contraction_scheme` can be used and is demonstrated below.
+
+```
+# Create a dummy circuit to test on.
+N = 5
+CX = [[1., 0., 0., 0.] [0., 1., 0., 0.] [0., 0., 0., 1.] [0., 0., 1., 0.]]
+tnc = TensorNetworkCircuit(N)
+for layer = 1:10, qubit = 1:N-1
+    push!(tnc, [qubit, qubit+1], CX)
+end
+add_input!(tnc)
+
+# Use FluwCutter for 10 seconds to find a contraction plan and then select 8 indices for slicing
+# using the tree trimming method.
+indices_to_slice, contraction_plan, metadata = contraction_scheme(tnc, 8; time=10, hypergraph=true)
+
+println("The number of indices to be sliced is ", length(indices_to_slice))
+println("The treewidth as a function of the number of indices sliced is ", 
+metadata["Slicing"]["Treewidths after slicing consecutive edges"])
+```
+
+## QXContexts output
+
+The output of a distributed simulation run by `QXContexts` is determined by output parameters in the simulation files generated by the `generate_simulation_files` function in `QXTools`. To dictate which output is generated by `QXContexts`, a dictionary containing paramters for the desired output method needs to be passed to `generate_simulation_files`. To this end, `QXTools` provides the function `output_params_dict` which will create the relevant dictionary for the desired output. The tree output methods available in QXContexts are described below. Note, for each method, `output_params_dict` takes two positional arguments: `num_qubits` and `num_outputs` giving the number of qubits in the circuit to be simulated and the desired number of outputs to generate (ie number of bitstring samples or number of amplitudes to compute).
+
+### Rejection
+
+To generate random bitstrings, which are distributed as if they were measured on a real quantum device running our quantum circuit, rejection sampling can be used. To create the output dictionary for rejection sampling, we can use the following keyword arguments of `output_params_dict`. 
```

Comment:
Wonder if saying real quantum device here might make some people think we are simulating the error processes. I guess we are in some way approximating. Maybe a few words to clarify would be useful.